nagios/tests/test_nagios.py

```diff
@@ -151,6 +144,71 @@ def test_create_event(self):
             'payload:DISK CRITICAL - free space: / 1499 MB (2.46% inode=77%):'
         ]
 
+    def test_event_message_title(self):
+        """
+        Check that right field is used as message title
+        """
+
+        events = [
+            {
+                'type': 'CURRENT HOST STATE',
+                'parts': ['domU-12-31-38-00-78-98', 'UP', 'HARD', '1', 'PING OK - Packet loss = 0%, RTA = 1.03 ms'],
+                'msg_title': 'CURRENT HOST STATE'
+            },
+            {
+                'type': 'CURRENT SERVICE STATE',
+                'parts': ['domU-12-31-38-00-78-98', 'Current Load', 'OK', 'HARD', '1', 'OK - load average: 0.04, 0.03'],
+                'msg_title': 'Current Load'
+            },
+            {
+                'type': 'SERVICE ALERT',
+                'parts': ['domU-12-31-39-02-ED-B2', 'cassandra JVM Heap', 'WARNING', 'SOFT', '1', ''],
+                'msg_title': 'cassandra JVM Heap'
+            },
+            {
+                'type': 'HOST ALERT',
+                'parts': ['domU-12-31-39-02-ED-B2', 'DOWN', 'SOFT', '1', 'PING CRITICAL - Packet loss = 100%'],
+                'msg_title': 'HOST ALERT'
+            },
+            {
+                'type': 'SERVICE NOTIFICATION',
+                'parts': ['pagerduty', 'ip-10-114-245-230', 'RAID EBS', 'OK', 'notify-service-by-email', ''],
+                'msg_title': 'RAID EBS'
+            },
+            {
+                'type': 'SERVICE FLAPPING ALERT',
+                'parts': ['domU-12-31-39-16-52-37', 'cassandra JVM Heap', 'STARTED', 'Service started flapping'],
+                'msg_title': 'cassandra JVM Heap'
+            },
+            {
+                'type': 'ACKNOWLEDGE_SVC_PROBLEM',
+                'parts': ['domU-12-31-39-16-52-37', 'NTP', '2', '1', '0', 'nagiosadmin', 'alq'],
+                'msg_title': 'NTP'
+            },
+            {
+                'type': 'HOST DOWNTIME ALERT',
+                'parts': ['ip-10-114-89-59', 'STARTED', 'Host has entered a period of scheduled downtime'],
+                'msg_title': 'HOST DOWNTIME ALERT'
+            },
+            {
+                'type': 'SERVICE DOWNTIME ALERT',
+                'parts': ['ip-10-114-237-165', 'intake', 'STARTED',
+                          'Service has entered a period of scheduled downtime'],
+                'msg_title': 'intake'
+            }
+        ]
+
+        for event in events:
+            self._check_event_msg_title(event_type=event['type'], parts=event['parts'], msg_title=event['msg_title'])
+
+    @staticmethod
+    def _check_event_msg_title(event_type, parts, msg_title):
```

Comment:
I would rename `msg_title` to `expected_msg_title`, will make it a bit more clear